### ISSUE TITLE
How to exit app?

### ISSUE THREAD

**Issue by pjebs** [VladimirMarkelov/clui#65]
I ran some of your demos.
I don't know how to exit.
Normally Control+C or Control+D exits app.

**Comment by VladimirMarkelov** [comment]
Yes, I did not use Ctrl+C or Ctrl+D - termbox demos do not use it as well. Ctrl+C is occupied - it is used to copy EditField text to clipboard. So, I used double Ctrl+Q(that is often GUI app default exit combination). I mentioned it in docs (Introduction part), maybe it is hard to spot at first site. Here is a doc quotation:

> Windows can be manipulated with both mouse and keyboard - the library provides built-in hotkeys to do common tasks like Window moving or resizing (and extra key sequence 'CtrlQ CtrlQ' to exit an application)

**Comment by pjebs** [comment]
For a mac, is Cmd+C used to copy text.
Currently Ctrl+C is copying text but that is not correct behaviour.

For a password protected editbox, the Copy text should be disabled for a mac.

**Comment by VladimirMarkelov** [comment]
> Currently Ctrl+C is copying text but that is not correct behaviour.

What if I add MacOS standard copy/paste combination to existing ones(Ctrl+C & Ctrl+V are often used in  Windows)? In this case, please, provide those combinations.

**Comment by pjebs** [comment]
Cut - Cmd+X
Copy - Cmd+C
Paste - Cmd+V

You can use build constraints for darwin at the top of the Go files.

On Mac for command line programs, Ctrl+C (Sig Interrupt which usually kills program) and Ctrl+D (EOF - intention to exit)

https://superuser.com/questions/169051/whats-the-difference-between-c-and-d-for-unix-mac-os-x-terminal

**Comment by pjebs** [comment]
what else do you want to know?

**Comment by VladimirMarkelov** [comment]
Oh. Sorry. It seems I got it wrong. You did not want to change copy/paste combination. You just wanted to disable copy for password protected fields? Is it correct? In this case I'll fix it tomorrow.

**Comment by pjebs** [comment]
For mac it should use Cmd+C for copy instead of Ctrl+C.
Also, disable copy for password protected fields

**Comment by VladimirMarkelov** [comment]
If you accept that I proposed in https://github.com/VladimirMarkelov/clui/issues/66 then I close this ticket and that one will be used for both issues

**Comment by pjebs** [comment]
https://electronjs.org/docs/api/accelerator
